src/util/cloud_utils.js

```diff
@@ -64,6 +64,22 @@ function get_azure_connection_string(params) {
     return connection_string;
 }
 
+function get_azure_new_connection_string(params) {
```

Comment:
Let's add a TODO that when there will no need for both functions (`get_azure_connection_string` and 
`get_azure_new_connection_string` ) we can remove the old and rename the new